javalib/src/main/scala/java/nio/charset/Charset.scala

```diff
@@ -8,6 +8,9 @@ abstract class Charset protected (canonicalName: String,
     aliases: Array[String]) extends AnyRef with Comparable[Charset] {
```

Comment:
Could you rename this to `_aliases` or `aliasesArray`, so that there isn't a semi-clash between the private `aliases` and the `def aliases()`?